marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils/invoke-queue-write.sjs

```diff
@@ -49,40 +46,18 @@ function deleteContent(content, temporalCollection) {
   }
 }
 
-/**
- * 
- * @param content 
- * @returns array of collections that the content object should be written to
- */
-function determineTargetCollections(content) {
-  const context = (content.context || {});
-  const contextCollections = context.collections || [];
-
-  let targetCollections;
-  if (context.useContextCollectionsOnly) {
-    targetCollections = fn.distinctValues(Sequence.from(contextCollections)).toArray();
-  } else {
-    targetCollections = fn.distinctValues(Sequence.from(configCollections.concat(contextCollections))).toArray();
-  }
-
-  return targetCollections.length > 0 ? targetCollections : xdmp.defaultCollections().toArray();
-}
-
 // Create a map of all temporal collections for quick checks on whether or not a collection is a temporal one
```

Comment:
Add a comment here about how we need to be careful about moving this, because this needs to run against the target database so it sees the correct temporal collections